srtp_cipher_test.go

```diff
@@ -591,6 +591,17 @@ func TestSrtpCipher(t *testing.T) {
 					assert.NoError(t, err)
 					assert.Equal(t, testCase.encryptedRTPPacket, actualEncrypted)
 				})
+
+				t.Run("Same buffer", func(t *testing.T) {
+					buffer := make([]byte, 0, 1000)
+					src, dst := buffer, buffer
+					src = append(src, testCase.decryptedRTPPacket...)
+					assert.True(t, isSameBuffer(dst, src))
+
+					actualEncrypted, err := ctx.EncryptRTP(dst, src, nil)
+					assert.NoError(t, err)
+					assert.Equal(t, testCase.encryptedRTPPacket, actualEncrypted)
+				})
```

Comment:
Maybe better to check `actualEncrypted` is also the same buffer of `src`

Reply:
Good point, added